src/main/java/bio/terra/service/dataset/DatasetService.java

```diff
@@ -667,7 +667,6 @@ private String writeIngestRowsToAzureStorageAccount(
               profile,
               storageAccount,
               tempFilePath,
-              AzureStorageAccountResource.ContainerType.SCRATCH,
```

Comment:
Yeah, so we can totally talk in person.  The things is that containers are kind of a special case. They're basically their own top level folder for a storage account that give you a little more ACL controls. So when we get a connection string to use Synapse, or upload files, we get a connection string that's based on container.

With everything in a dataset now living in the same container, there's no need to specify a different container since all the folders have the same connection string.